ui/desktop/electron-app/src/services/origin.js

```diff
@@ -0,0 +1,8 @@
+const RuntimeSettings = require('../models/runtime-settings.js');
+
+// Provides a singleton class instance to enable a consistent view of
```

Comment:
Runtime settings is more general than the "origin" filename would suggest.

Reply:
Runtime settings is more of a service.

Reply:
This was a bit confusing - as to why did we name it `origin` in the first place. I'll update it to `runtimeSettings` service for future sanity.

Reply:
Updated.